ocs_ci/deployment/deployment.py

```diff
@@ -646,6 +649,109 @@ def patch_default_sc_to_non_default(self):
+        # Edit the configs.imageregistry.operator.openshift.io object and remove the content in the storage section.
+        ir_obj = ocp.OCP(namespace=constants.OPENSHIFT_IMAGE_REGISTRY_NAMESPACE)
+        if self.platform == constants.AWS_PLATFORM:
+            ir_obj.patch(resource_name='configs.imageregistry.operator.openshift.io',
+                         params=" '{\"spec\":{\"storage\":{}}}' ")
+
+        elif self.platform == constants.VSPHERE_PLATFORM:
+            ir_obj.patch(resource_name='configs.imageregistry.operator.openshift.io',
+                         params=" '{\"spec\":{\"storage\":{\"emptyDir\":{}}}}' ")
+
+        # Removing the cluster logging operator from OpenShift Container Storage
+        # Remove the ClusterLogging instance in the namespace.
+        ocp_obj.exec_oc_cmd("delete clusterlogging instance -n openshift-logging --wait=true --timeout=5m")
+
+        # STEP3: Follow these instructions to ensure that the PVCs listed in the previous step are deleted
+        # STEP 3.1: Determine the pod that is consuming the PVC.
+        pod_list = get_all_pods()
+        for pod in pod_list:
+            for pvc in pvcs_to_delete:
+                if get_pvc_name(pod) == pvc.get('metadata').get('name'):
+                    pod.delete()
+
+        # STEP 3.2: Delete the PVCs.
+        for pvc in pvc_list:
+            pvc.delete()
+
+        # STEP 4: Delete the StorageCluster object.
+        storage_cluster = ocp.OCP(kind=constants.STORAGECLUSTER, resource_name="ocs-storagecluster")
+        storage_cluster.delete()
+
+        # STEP 5: Delete the namespaces and wait till the deletion is complete.
+        oc = openshift_ops.OCP()
+        project_list = oc.get_projects()
+        for project in project_list:
+            if "test" in project.get('metadata').get('name'):
+                ocp_obj.delete_project(project.get('metadata').get('name'))
+        ocp_obj.delete_project('openshift-storge')
+
+        # STEP 6: List the storage nodes.
+        nodes_list = get_labeled_nodes('cluster.ocs.openshift.io/openshift-storage')
+
+        # STEP 7: For each node, perform the following:
+        for node in nodes_list:
+            ocp_obj.exec_oc_debug_cmd(node=node, cmd_list=["rm -rf /var/lib/rook"])
+
+        # STEP 8: Delete the storage classes with an openshift-storage provisioner listed in step 1.
+        delete_storageclasses(sc_list)
+
+        # STEP 9: Remove the taint from the storage nodes.NOT SUPPORTED YET
+        """ ocp_obj.exec_oc_cmd("adm taint nodes --all node.ocs.openshift.io/storage-") """
+
+        # STEP 10: Unlabel the storage nodes.
+        nodes_list.append(get_labeled_nodes('topology.rook.io/rack'))
+        for node in nodes_list:
+            ocp_obj.add_label(resource_name=node, label='cluster.ocs.openshift.io/openshift-storage-')
+            ocp_obj.add_label(resource_name=node, label='topology.rook.io/rack-')
```

Comment:
I think this is needed only in vsphere

Reply:
when running unlable manually in CMD this doesn't throw any error, just skipping if the label wasn't found, I dont think It bothers